packages/components/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 7.2.0 (Unreleased)
+
+### New Features
+
+- `ToggleControl` now accepts a `className` prop to merge with its default-assigned classname.
```

Comment:
> I'm no longer sure what is the correct one :)

Funny you should mention this, as I had the same dilemma occur to me yesterday in thinking about a separate task. Since `className` is camel-cased, it seems like it'd be two separate words written out, but I think we're so accustomed to seeing it as one unit that it looks not-quite-right.